app/views/labs/index.html.erb

```diff
@@ -14,7 +14,6 @@
             tags = Array.new
             lab.lab_spaces.each do | ls | ls.equipment.each do | e |
```

Comment:
Could we write each loop in it's own line? I think it would be clearer for anyone who reads it.